exotica/Vector q0
exotica/Vector qT
geometry_msgs/Pose base_pose
moveit_msgs/PlanningScene ps
---
bool succeed
exotica/Vector[] solution
---
uint32[] solution_indices

